dd-trace-ot/src/test/groovy/datadog/opentracing/decorators/URLAsResourceNameTest.groovy

```diff
@@ -82,6 +82,12 @@ class URLAsResourceNameTest extends DDSpecification {
     "/V01/v9/abc/-1"                                   | "/V01/v9/abc/?"
     "/ABC/av-1/b_2/c.3/d4d/v5f/v699/7"                 | "/ABC/?/?/?/?/?/?/?"
     "/user/asdf123/repository/01234567-9ABC-DEF0-1234" | "/user/?/repository/?"
+    "/user/1/repo/test@example.com/"                   | "/user/?/repo/?/"
+    "/user123@host-456.biz"                            | "/?"
+    "/a@b@c"                                           | "/?"
+    "/@nothing-before-at.com"                          | "/?"
+    "/endsinat@/includedsegment"                       | "/?/includedsegment"
+    "/@"                                               | "/?"
```

Comment:
We'll need a modified header for this file

Reply:
Nikita: this just means placing the string
// Modified by SignalFx
at the top of the file.  We do this for all files we modify from upstream.